### DISCUSSION TITLE
[求助] 怎么快速成为传奇绳匠

### DISCUSSION THREAD

**Discussion by Anuan-shu** [share121/inter-knot#2807] [其他]
<img src="https://bbs-static.miyoushe.com/static/2023/11/17/e0564870de2f3d1c39dfc41a8dcd9d2b_383241188874318913.png" alt="芙宁娜-好困难" width="75" >
之前的账号被封了，现在改用小号，怎么快速升级成传奇绳匠啊<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by bangbu01** [comment]
你是法厄同？

**Comment by cx525** [comment]
v我50告诉你<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by Aha-SWYH** [comment]
当过传奇绳匠的人一定不会问吧？

**Comment by BITE-7274** [comment]
创建仓库即可·-·

**Comment by TetsuyaTetsuya** [comment]
请问是新艾利都六分街randomplay录像店的哲铃兄妹吗（doge

**Reply by NEWERIDU** [reply]
我超，盒（

**Comment by Lanxey** [comment]
找本主角重活一世的网文参考

**Comment by KEARUM** [comment]
v我50菲林就告诉你

**Comment by aizai1173** [comment]
V我50，复制一个Fariy给你，电费自负<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by trifortune** [comment]
我这里有份前文明的资料可以参考一下 `https://fanqienovel.com/page/7400742792831437886`

**Comment by linlu0792** [comment]
我虽然不是专业的，但我听说只要把绳网上所有的委托都横扫一遍，就可以成为传奇绳匠了，每一个传奇绳匠都是在全网委托人的见证下诞生的，因为在这一段时间所有人的委托都被同一个账户完成了，大家为表示尊敬纷纷欢呼，共襄盛举的场面被后人冠以“传奇”之名
当然，如果你做到了这些却没有成为传奇绳匠那当我没说，毕竟我也不是专业的<img src="https://bbs-static.miyoushe.com/static/2024/02/02/abc8f12e02fe2104201d8ae615e5d78b_3586427210969548681.png" alt="寻梦者-让我想想" width="75" >

**Comment by waerte-yang** [comment]
这个委托我接了！

**Comment by yzysrewrwqr** [comment]
这个委托楼上接了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Pheathon** [comment]
这个委托我接了。

**Comment by yivitte** [comment]
疑似法厄同小号的账号增加了，回头找联络人启明星查查你的业绩

**Comment by XuemeiLing** [comment]
这个委托我接了！

**Comment by wuyueji** [comment]
铃，你哥哲在我手里（

**Comment by NEWERIDU** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >哼哼
